### ISSUE TITLE
Running plugins in Tissuumaps server mode

### ISSUE THREAD

**Issue by Mena-SA-Kamel** [TissUUmaps/TissUUmapsCore#15]
Hi!

This is not really an issue, but rather a question. Where can I find instructions on how to load plugins into the server-based web viewer?
I really like the ["FeatureSpace" plugin](https://vimeo.com/685804904) and would like to use it on a Tissuemaps instance running on a server instead of locally.  Please let me know if this is possible.

Thanks
Mena

**Comment by Mena-SA-Kamel** [comment]
Yes, I meant a server running TissUUmaps server using `tissuumaps_server`

**Comment by cavenel** [comment]
Then you can copy these two files from the `USER_DIRECTORY/.tissuumaps/plugins/` folder of your local computer to the `USER_DIRECTORY/.tissuumaps/plugins/` folder on the server:
 * Feature_Space.js
 * Feature_Space.py

If you don't have these files locally you can download them on https://tissuumaps.github.io/TissUUmaps/plugins/
(The `tissuumaps_server` command should display the "Plugins" menu as long as an image or a project is open)

**Comment by Mena-SA-Kamel** [comment]
Okay perfect! I will give that a try. Thanks a lot.

Mena

**Comment by cavenel** [comment]
Great, let me know if it works, so that I can close the issue :)
Also note that from version [3.0.10.3](https://github.com/TissUUmaps/TissUUmaps/releases/tag/3.0.10.3), plugins will be visible from the home page in the server mode, even without any opened image or project.

**Comment by cavenel** [comment]
Hi,

Great, if it works then I will close this issue.

For the closing button of the Feature Space plugin, you can update to v.1.1 of the FeatureSpace plugin and it should work now (https://tissuumaps.github.io/TissUUmaps/plugins/)

We can discuss the other points in separate issues, especially the one about saving plugin states in the tmap file, which is something I've been wanting to add for some time already but is not possible yet :(
